ci-operator/config/openshift/os/openshift-os-master__okd-scos.yaml

```diff
@@ -3,6 +3,11 @@ base_images:
     name: builder
     namespace: ocp
     tag: stream9
+build_root:
```

Comment:
@jlebon looks like specifying the build_root image is necessary- it shouldn't matter what we use here, but i just used coreos-assembler for consistency with the other configs